app/models/dish.rb

```diff
@@ -18,16 +18,24 @@ class Dish < ActiveRecord::Base
   RATE_DIMENSION_DISH = 'dish'
   ratyrate_rateable RATE_DIMENSION_DISH
 
-  def rates
+  # public activity
+  include PublicActivity::Model
+  tracked owner: Proc.new{ |controller, model| controller.try(:current_user) }, recipient: Proc.new{ |controller, model| model.try(:user) } 
```

Comment:
As that method accepts symbols (which then get called as methods on the current model), I think the `recipient` option could be simplified so the `tracked` line becomes:
```ruby
tracked owner: Proc.new{ |controller, model| controller.try(:current_user) }, recipient: :user 
```